rest/replicatortest/replicator_conflict_test.go

```diff
@@ -1526,8 +1780,403 @@ func TestActiveReplicatorHLVConflictCustom(t *testing.T) {
 				assert.Equal(t, resolvedDocBodyLocal, resolvedDocBodyRemote)
 			}
 
-			// todo: assert on rev tree here - CBG-4791
+			if testCase.newCVGenerated {
+				// merge/local wins will:
+				//	- tombstones local active revision
+				//  - winning merged rev is written as child of remote winning rev
+				remoteGeneration, _ := db.ParseRevID(ctx1, remoteDoc.GetRevTreeID())
+				newRevID := db.CreateRevIDWithBytes(remoteGeneration+1, remoteDoc.GetRevTreeID(), []byte(testCase.expectedBody))
+				require.NoError(t, err)
```

Comment:
Invalid error check - maybe `require.NotEmpty` if anything?

```suggestion
				require.NotEmpty(t, newRevID)
```